website/pages/docs/agent/options.mdx

```diff
@@ -1930,13 +1931,15 @@ Reloading configuration does not reload all configuration items. The
 items which are reloaded include:
 
 - ACL Tokens
+- [Bootstrap](#config_entries_bootstrap)
```

Comment:
```suggestion
- [Configuration Entry Bootstrap](#config_entries_bootstrap)
```

Reply:
Will update, but can I ask what other types of bootstrapping there are that need this granularity?

Reply:
There are at least 2 other bootstraps in Consul that I was able to find: one for electing an initial server leader, and another for ACLs